pkg/build/builder/util_linux.go

```diff
@@ -98,3 +100,14 @@ func getCgroupParent() (string, error) {
 	glog.V(6).Infof("found cgroup values map: %v", cgMap)
 	return extractParentFromCgroupMap(cgMap)
 }
+
+// getSELinuxLabel retrieves the selinux label associated w/ this container.
+func getSELinuxLabel() (string, error) {
+	v, e := selinux.CurrentLabel()
+	if e != nil {
+		return "", e
+	}
+	// convert unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023
+	// into s0-s0:c0.c1023
+	return strings.SplitN(v, ":", 4)[3], nil
```

Comment:
@rhatdan is this what you were proposing?  if not i'm going to need a few more hints.  thanks.

Reply:
@rhatdan this is returning a value w/ some spurious trailing character:

`found selinux label: unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023^@`

i'm happy to trim it, but first i'd like some confirmation as to why it's there.

Reply:
This would happen if you ran the code in a test framework, because it is showing the label of the process running it.  Most likely you are not in a container running this command so it is showing you a user label. unconfined_t.
If you run it inside of a container you should see something like svirt_lxc_net_t, or container_t.

Reply:
it's being run from within a container.

Reply:
If this is being run within a container then someone is forcing this to happen.  Containers and processes launched by the init system or a container runtime should not be running as unconfined_t.  

Did someone setup a unit file to run as unconfined_t or specify at label to run continares as unconfined_t?